packages/trie/src/types.ts

```diff
@@ -32,6 +32,8 @@ export interface TrieOpts {
   /**
    * Delete nodes from DB on delete operations (disallows switching to an older state root)
    * Default: `false`
+   * Note: this only deletes final leaf nodes from the DB. For more active pruning
+   * (deleting all unreachable key hashes from the DB), use `useNodePruning`.
```

Comment:
In a couple of instances it is written, in comments:
`(delete operations are only executed on DB with `deleteFromDB` set to `true`)`
I don't think that's true anymore with `useNodePruning`